baker/algolia/configureAlgolia.ts

```diff
@@ -109,6 +109,23 @@ export const configureAlgolia = async () => {
         disableExactOnAttributes: ["tags"],
     })
 
+    const explorersIndex = client.initIndex("explorers")
+
+    await explorersIndex.setSettings({
+        ...baseSettings,
+        searchableAttributes: [
+            "unordered(slug)",
+            "unordered(title)",
+            "unordered(subtitle)",
+            "unordered(views_7d)",
+            "unordered(text)",
+        ],
+        customRanking: ["desc(score)", "desc(importance)"],
```

Comment:
Both of these don't exist on explorers:

```suggestion
        customRanking: ["desc(vies_7d)"],
```